package.json

```diff
@@ -577,9 +581,9 @@
 		"watch": "tsc -watch -p ./",
 		"pretest": "yarn run compile && yarn run lint && yarn run esbuild",
 		"lint": "eslint src --ext ts",
-		"test": "env CURSORLESS_TEST=true node ./out/test/runTest.js",
+		"test": "env CURSORLESS_TEST=true node ./out/test/scripts/runTestsCI.js",
```

Comment:
I tried this, but it broke, because `npm test` is a bit special; it runs `pretest`.  I figure better not to mess with it, but lmk if you feel strongly @AndreasArvidsson